src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -762,6 +842,30 @@ public int deleteMarkedForDelete(String origin) throws Exception {
     }
 
     
+    /**
+     * 
+     * Update a mapping new a new kalturaId
+     * 
+     * @param mappingDto
+     * @throws Exception Will throw exception if id is not found
+     */     
+    public void updateMapping(MappingDto mappingDto) throws Exception {
+       // Sanity check
+        if (mappingDto.getReferenceId() == null) {
+            throw new InvalidArgumentServiceException("Id must not be null"); 
```

Comment:
Again, please specify which ID cant be null

Reply:
fixed